README.md

```diff
@@ -45,39 +55,35 @@ As you can see below, without increasing the footprint of these badges, I've tri
 This badge design corresponds to an old and now deprecated version which has since been replaced by beautiful and scalable SVG versions that can be found on [shields.io](http://shields.io).
 
 ## Examples
-
-What kind of meta data can you convey using badges?
-
-- test build status: `build | failing`
-- code coverage percentage: `coverage | 80%`
-- stable release version: `version | 1.2.3`
-- package manager release: `gem | 1.2.3`
-- status of third-party dependencies: `dependencies | out-of-date`
-- static code analysis GPA: `code climate | 3.8`
-- [semver](http://semver.org/) version observance: `semver | 2.0.0`
-- amount of [gratipay](http://gratipay.com) donations per week: `tips | $2/week`
-
+What kind of metadata can you convey using badges?
+* test build status: `build | failing`
+* code coverage percentage: `coverage | 80%`
+* stable release version: `version | 1.2.3`
+* package manager release: `gem | 1.2.3`
+* status of third-party dependencies: `dependencies | out-of-date`
+* static code analysis GPA: `code climate | 3.8`
+* [SemVer](http://semver.org/) version observance: `semver | 2.0.0`
+* amount of [Gratipay](http://gratipay.com) donations per week: `tips | $2/week`
 
 ## Services using the Shields standard
-- [Travis CI](https://github.com/travis-ci/travis-ci/issues/630#issuecomment-38054967)
-- [Code Climate](https://codeclimate.com/changelog/510d4fde56b102523a0004bf)
-- [Coveralls](https://coveralls.io/r/kaize/nastachku)
-- [Gemfury/RubyGems](http://badge.fury.io/)
-- [Gemnasium](http://support.gemnasium.com/forums/236528-general/suggestions/5518400-use-svg-for-badges-so-they-still-look-sharp-on-r)
-- [GoDoc](https://godoc.org/)
-- [Scrutinizer CI](https://scrutinizer-ci.com/)
-- [Semaphore](https://semaphoreapp.com)
-- [VersionEye][]
-- [Read the Docs](https://readthedocs.org/)
-- [BadgerBadgerBadger][gem]
-- [badges2svg][]
-- [reposs][]
-- [ruby-gem-downloads-badge][]
-- [PHPPackages](https://phppackages.org)
+* [Badger][gem]
+* [badges2svg][]
+* [Code Climate](https://codeclimate.com/changelog/510d4fde56b102523a0004bf)
+* [Coveralls](https://coveralls.io/)
+* [Gemnasium](http://support.gemnasium.com/forums/236528-general/suggestions/5518400-use-svg-for-badges-so-they-still-look-sharp-on-r)
+* [GoDoc](https://godoc.org/)
+* [PHPPackages](https://phppackages.org)
+* [Read the Docs](https://readthedocs.org/)
+* [reposs][]
+* [ruby-gem-downloads-badge][]
+* [Scrutinizer](https://scrutinizer-ci.com/)
+* [Semaphore](https://semaphoreapp.com)
+* [Travis CI](https://github.com/travis-ci/travis-ci/issues/630#issuecomment-38054967)
+* [Version Badge](http://badge.fury.io/)
+* [VersionEye]
 
 [gem]: https://github.com/badges/badgerbadgerbadger
```

Comment:
The format for URL linking is a little inconsistent. Perhaps you should place them all inline instead of at the bottom?